experiment/esm-scen7-h-aer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "esm-scen7-h-aer",
+    "validation-key": "esm-scen7-h-Aer",
+    "ui-label": "Future scenario esm-scen7-h with high aerosol emissions",
+    "description": "1. Detailed experiment configuration Future scenario esm-scen7-h with higher aerosol emissions  2. Required model settings We encourage modelling centres to include as much atmospheric composition capability as possible. As a minimum, models should have prescribed or interactive aerosols. And the experimental setup should be consistent with other scenario experiments.  3. Experiment conditions  4. Links to relevant references Fiedler et al. (in prep.) AerChemMIP2 - Unraveling the role of reactive gases, aerosols, and land use for air quality and climate change in CMIP7  5. Similarities to CMIP6 experiments None  6. Forcing differences from parent experiment Increased aerosol emissions compared to parent.",
```

Comment:
(Not your issue @matthew-mizielinski, but) This description is quite vague. I would be tempted to go back to the AerChemMIP chairs to see if they want to quantify what 'higher' means exactly and which forcing dataset modelling teams should use for this before merging.